specs/waterOrderingSpec.js

```diff
@@ -0,0 +1,43 @@
+const assert = require('assert')
+const rewiremock = require('rewiremock/node');
+
+rewiremock('@sendgrid/mail').with(sendgridMock())
+rewiremock('https').callThrough().with(httpsGetMock())
+
+rewiremock.enable()
+const waterOrderingController = require('../features/waterOrdering')
+rewiremock.disable()
+
+const replies = require('../features/waterOrdering/replies.js')
+const { getBasicController } = require('./helpers')
+
+function sendgridMock() {
+  return ({
+    send: async (message) => {},
+    setApiKey: (key) => {}
+  })
+}
+
+function httpsGetMock() {
+  return ({
+    get: async (url) => {}
+  })
+}
```

Comment:
@npupko BTW, what do you think about moving them to helpers (`helpers/mocks` folder? 🤔) It looks like useful things for future specs

Reply:
@npupko Yes, but let's use `specs/mocks` instead of `specs/helpers/mocks`?

Reply:
@npupko Sounds good 👍